SQL/New_patches/2019_07_04_remove_header_row_from_parameter_file_and_convert_back_to_Value_field_to_text.sql

```diff
@@ -0,0 +1,3 @@
+DELETE FROM parameter_file WHERE ParameterTypeID=(SELECT ParameterTypeID FROM parameter_type WHERE Name='header' AND SourceFrom='parameter_file');
+DELETE FROM parameter_type WHERE Name='header' AND SourceFrom='parameter_file';
+ALTER TABLE parameter_file MODIFY Value TEXT;
```

Comment:
@cmadjar I don't know the kind of value that are there but any value longer that 64KB will be trunct to the 1st 64KB.

If there should not be any value longer than that after the delete, I approve this PR

Reply:
@PapillonMcGill the insertion script is filtering any value longer than 1000 characters so moving it back to text is fine. The only exception was the 'header' which was crazy long and that we are getting rid of in this PR.

Reply:
Thanks for the clarification.